javascript-primitives-and-variables.tar/javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,10 @@
+const fullName = 'Lambert Nguyen';
+const isCool = true;
+const totalPets = 1;
+console.log('fullName: ', fullName, typeof fullName);
+console.log('isCool: ', isCool, typeof isCool);
+console.log('totalPets: ', totalPets, typeof totalPets);
+const unknown = null;
+console.log('unknown: ', unknown, typeof unknown);
+const never = undefined;
```

Comment:
just a heads up - the instructions state to not assign `never` a value.

The end result is the same. If you had typed `const never;` it would still be `undefined` but you should avoid explicitly assigning `undefined` to variables when possible.